rocAL_pybind/amd/rocal/readers.py

```diff
@@ -78,7 +78,7 @@ def file(file_root, file_filters=None, file_list='', stick_to_shard=False, pad_l
     Pipeline._current_pipeline._reader = "labelReader"
     # Output
     labels = []
-    kwargs_pybind = {"source_path": file_root}
+    kwargs_pybind = {"source_path": file_root, "file_list": file_list}
```

Comment:
just send the correct path here itself

Reply:
Both source path and file_list_path are passed as seperate args from user. Source path is used only when file_list_path passed is an empty string. Also user can pass relative file paths in file list and we use source path as the root folder for the relative paths.